dts/arm/renesas/smartbond/da1469x.dtsi

```diff
@@ -19,6 +20,66 @@
 		};
 	};
 
+	crg {
+		osc: osc {
+			compatible = "renesas,smartbond-clock";
+			status = "okay";
+			rc32k: rc32k {
+				compatible = "fixed-clock";
+				clock-frequency = <DT_FREQ_K(32)>;
+				#clock-cells = <0>;
+				status = "okay";
+			};
+			xtal32k: xtal32k {
+				compatible = "fixed-clock";
+				clock-frequency = <32768>;
+				#clock-cells = <0>;
+				status = "disabled";
+			};
+			rcx: rcx {
+				compatible = "fixed-clock";
+				clock-frequency = <DT_FREQ_K(15)>;
+				#clock-cells = <0>;
+				status = "disabled";
+			};
+			rc32m: rc32m {
+				compatible = "fixed-clock";
+				clock-frequency = <DT_FREQ_M(32)>;
+				#clock-cells = <0>;
+				status = "okay";
+			};
+			xtal32m: xtal32m {
+				compatible = "fixed-clock";
+				clock-frequency = <DT_FREQ_M(32)>;
+				#clock-cells = <0>;
+				status = "disabled";
+			};
+			pll: pll {
+				compatible = "fixed-clock";
+				clock-frequency = <DT_FREQ_M(96)>;
+				#clock-cells = <0>;
+				status = "disabled";
```

Comment:
We should state the dependency between PLL and XTAL32M. Right now if xtal32m status is disabled and pll status is ok, we could enable PLL in `smartbond_clocks_init()` and disable XTAL32M.